cmd/helm.go

```diff
@@ -144,7 +146,14 @@ func (d *diffCmd) vals() ([]byte, error) {
 	// User specified a values files via -f/--values
 	for _, filePath := range d.valueFiles {
 		currentMap := map[string]interface{}{}
-		bytes, err := ioutil.ReadFile(filePath)
+
+		var bytes []byte
+ 		var err error
+ 		if strings.TrimSpace(filePath) == "-" {
+ 			bytes, err = ioutil.ReadAll(os.Stdin)
+ 		} else {
+ 			bytes, err = ioutil.ReadFile(filePath)
+ 		}
```

Comment:
For reference: https://github.com/helm/helm/blob/581e6cdbb8e42753556ae48ccba8cef84c40f736/cmd/helm/install.go#L373-L379